MetaLamp/lending-pool/client/package.json

```diff
@@ -26,8 +26,10 @@
     "jquery": "^3.3.1",
     "json-bigint": "^1.0.0",
     "node-sass": "^4.12.0",
+    "purescript": "^0.13.6",
```

Comment:
Actually if I remove these dependencies, everything seems to work.

Reply:
@stanislav-az can you please try to test client too without `purescript` and `spago` in `package.json`? Does it work for you?

Reply:
@olgaklimenko it works for me

Reply:
@olgaklimenko removing `purescript` and `spago` from `package.json` works too though.